pkg/ui/src/views/statements/statementDetails.tsx

```diff
@@ -505,9 +505,12 @@ export class StatementDetails extends React.Component<StatementDetailsProps, Sta
                   format: Bytes,
                 },
               ].filter(function (r) {
-                if (r.name === "Network Bytes Sent" && r.value.mean === 0) {
-                  // Omit if empty.
-                  return false;
+                if (r.name === "Network Bytes Sent" && r.value) {
```

Comment:
Should we instead make sure the stat is always populated on the server side? This feels fine but I don't like how unprincipled the protobufs are - it's like any value could be present or not :/

@dhartunian do you have thoughts here?

Reply:
I'm not sure I understand the suggestion. The protobuf field is non-nullable and it is valid for it not to be populated (until always-on tracing is a thing).

Reply:
Hmm, okay. I guess I'm the one who doesn't understand!

Reply:
Let me answer your questions. There is still one unknown here which is why is `r.value` undefined in the first place (https://github.com/cockroachdb/cockroach/issues/56432). I don't have an answer to that yet.

Reply:
@dhartunian has a PR resolving the undefined issue: https://github.com/cockroachdb/cockroach/pull/56445/files. Seems like the PR also resolves the issue this PR tries to fix (specifically accessing an undefined value), so will close this in favor of that one. Let me know if you want to discuss anything else related to this.